server/admin_management/api/app/routers/router_guards.py

```diff
@@ -132,7 +132,7 @@ async def enforce_self_grant_guard(
                 status_code=HTTPStatus.FORBIDDEN,
                 detail={
                     "code": ERROR_SELF_GRANT_PROHIBITED,
-                    "description": "Altering permission privilege to self is not allowed",
+                    "description": "Granting roles to self is not allowed.",
```

Comment:
Hi Jota, you probably missed, but as already mentioned in the ticket, currently you can fix this in frontend.
We will need a good approach for both frontend/backend to show consistent messages and will refactor that time for backend if needed. (please discard both `server/admin_management/api/app/routers/router_guards.py` and `server/backend/api/app/routers/router_guards.py` change)

Reply:
hello, Ian! I'll change the fix to be in frontend